@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1197,185 +1182,6 @@ export class TileGeometryCreator {
         }
     }
 
-    /**
-     * Gets the appropriate [[TextRenderStyle]] to use for a label. Depends heavily on the label's
-     * [[Technique]] and the current zoomLevel.
-     *
-     * @param technique Label's technique.
-     * @param techniqueIdx Label's technique index.
-     */
-    getRenderStyle(
-        tile: Tile,
```

Comment:
These two functions below getRenderStyle / getLayoutStyle were duplicated in PoiManager. I've moved them to TextStyleCache.